README.md

```diff
@@ -3,3 +3,28 @@
 | Windows                                                                                                                                                                           | Mac / Linux                                                                                                                                                     |
 | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | [![Build history](https://buildstats.info/appveyor/chart/YoloDev/yolodev-expecto-testsdk?branch=master)](https://ci.appveyor.com/project/YoloDev/yolodev-expecto-testsdk/history) | [![Build history](https://buildstats.info/travisci/chart/YoloDev/YoloDev.Expecto.TestSdk?branch=master)](https://travis-ci.org/YoloDev/YoloDev.Expecto.TestSdk) |
+
+
+## Configuration
+
+You can configure some of Expecto via `dotnet test`. `dotnet test` allows you to pass in `RunSettings` via the [CLI](#dotnet-test-runsettings) or using a [.runsettings](https://docs.microsoft.com/en-us/visualstudio/test/configure-unit-tests-by-using-a-dot-runsettings-file?view=vs-2017#example-runsettings-file) file. 
+
+### dotnet test RunSettings 
+
+To familiarize yourself with how dotnet test expects the CLI args, the help from it is shown below.
+
+```
```

Comment:
This should probably be a quote section, instead of a code-section.

Reply:
👍